.github/workflows/CI.yml

```diff
@@ -193,6 +193,17 @@ jobs:
           override: true
       - run: cargo build --features=c-interface
 
+  serde:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - run: cargo build --features=serde
```

Comment:
I think it'd be better to run it on all platforms. So please add this check after the "Run CPU test".